test_regress/t/t_cover_line.out

```diff
@@ -138,10 +138,10 @@
                  end
 %000000          do ; while (0);
 -000000  point: comment=block
-%000000          do begin
+ 000010          do begin
 +000010  point: comment=if
 -000000  point: comment=block
-%000000             $write("");
+ 000010             $write("");
```

Comment:
It looks like max fixed this logical inconsistency as well? I don't think there's any world where min is correct here